src/cargo/util/command_prelude.rs

```diff
@@ -1248,15 +1248,18 @@ fn get_target_triples() -> Vec<clap_complete::CompletionCandidate> {
     let mut candidates = Vec::new();
 
     if let Ok(targets) = get_target_triples_from_rustup() {
-        candidates = targets;
+        candidates.extend(targets);
     }
 
     if candidates.is_empty() {
         if let Ok(targets) = get_target_triples_from_rustc() {
-            candidates = targets;
+            candidates.extend(targets);
```

Comment:
why this change?

Reply:
I believe I made that change incidentally, and forgot to revert it. I will revert it.